### PR TITLE
Pits 2024 Edition

### PR DESCRIPTION
**Pull request by dominic-03**
New Java-based application for PITS, has only download functionality but commit and configure to come later. Developed and tested on a MacBook, might need extra testing on other operating systems (Windows 10/11, Linux if desired)

### CODE REVIEW

**Review by kryllyxofficial01 — APPROVED** [review]
looks good to me

File: .gitignore
Comment (NotMePipe) [inline .gitignore:1]: You may want to bring back the log directory to the gitignore since it is being created if it does not exist

File: README.md
Comment (NotMePipe) [inline README.md:1]: Not required, but it’d be nice to have a README for future years to know how to use this

**Review by NotMePipe — APPROVED** [review]
We’ve already discussed the features we still need/strongly want PITS to have. I made a few notes, but it looks good

**Review by dracco1993 — APPROVED** [review]

**Pull request merged**